### DISCUSSION TITLE
Chakra UI & Tailwind conflict

### DISCUSSION THREAD

**Discussion by chakradarraju** [chakra-ui/chakra-ui#9379] [General]
I had faced this conflict multiple times and this stackoverflow answer solved my problem: https://stackoverflow.com/questions/76421215/background-color-of-chakra-ui-solid-button-disappear

Essentially following snippet needs to be added to `tailwind.config.js` to avoid chakra UI style breaking.
```
  corePlugins: {
    preflight: false,
  }
```

Can we include this in install instruction?

**Comment by segunadebayo** [comment]
Chakra's reset and Tailwind Preflight both set base styles. Turn Preflight off:

```js
// tailwind.config.js
corePlugins: { preflight: false }
```

Or scope Tailwind so it does not hit Chakra nodes.
